test/functional/persistence/entity-updation/persistence-entity-updation.ts

```diff
@@ -28,7 +28,7 @@ describe("persistence > entity updation", () => {
         const post = new PostUuid();
         post.text = "Hello Post";
         await connection.manager.save(post);
-        const loadedPost = await connection.manager.findOne(PostUuid);
+        const loadedPost = await connection.manager.findOne(PostUuid, post.id);
```

Comment:
There are several tests that I fixed like this. I'm not sure if this is your preferred fix, though. findOne() now throws an error if no findOptions are specified.